### ISSUE TITLE
Social Media Graphics

### ISSUE THREAD

**Issue by carynaudenried** [bethlakshmi/gbe-divio-djangocms-python2.7#518]
Creating a thread here to review and share social media graphics.

**Comment by carynaudenried** [comment]
<img width="540" alt="ApplicationsOpen" src="https://github.com/bethlakshmi/gbe-divio-djangocms-python2.7/assets/112599862/5ab4328c-a6d7-4576-b7b1-6da7a38e43d3">
@burlexpo all the info look correct on here? This would be a good one to post before Oct. 1 to build buzz.

This is a quick one with the website graphics, I'll just keep making more and will push out to our volunteer team to share when we want to start promoting. If you have any requests for versions, just tag me here with what you want!

**Comment by burlexpo** [comment]
All info is correct! I'll share it out to the troops!

**Comment by burlexpo** [comment]
@carynaudenried Some other useful info:
Applications are "Pay What You Can" until October 31
Applications are open until Jan. 15, 2024
Notifications go out Feb 1, 2024

I think that the witch says "Salem, _Massachusetts_" pretty clearly, but there is a Salem in Oregon (and New Hampshire).  Do we care?

**Comment by carynaudenried** [comment]
@burlexpo Yeah good call about Salem. I think some of that info could be saved for a caption but I'll see how it fit in here!

Regardless I'll update the location on this one, and I'll also create some other options that have a bit more detail in them so we can have some options and post multiple times!

**Comment by carynaudenried** [comment]
@burlexpo Ok I obviously lost track of time with the last one, but here's another graphic that could be posted tomorrow to announce apps opening. Doesn't have the website on there, so it'd need to have that in the caption wherever it's posted. 

<img width="540" alt="ApplicationsOpen10 1 23" src="https://github.com/bethlakshmi/gbe-divio-djangocms-python2.7/assets/112599862/a8fb46ea-72df-485d-8826-e2cffa9d8d40">

**Comment by burlexpo** [comment]
Super! Will do!

**Comment by bethlakshmi** [comment]
Is this ticket good to close?  It looks like a happy ending?

**Comment by burlexpo** [comment]
@bethlakshmi We should probably create some more as applications get closer to closing.  

@carynaudenried can you alter the text on those to "Applications Now Open!" or something similar?
